src/modules/organization/InternalGroups.tsx

```diff
@@ -32,75 +25,66 @@ const InternalGroupsContainerTitle = styled.h2`
+  ${props => props.theme.media.mobile} {
     grid-template-columns: 1fr;
   }
   gap: 10px;
   border-radius: 4px;
+  margin-bottom: 1.618em;
 `
 
 interface AllInternalGroupsReturns {
-  allInternalWorkGroups: InternalGroupNode[]
-}
-
-interface AllInterestGroupsReturns {
-  allInterestGroups: InternalGroupNode[]
+  internalGroups: InternalGroupNode[]
+  interestGroups: InternalGroupNode[]
 }
 
 export const InternalGroups: React.VFC = () => {
   const history = useHistory()
-  const {
-    loading: internalLoading,
-    error: internalError,
-    data: internalData,
-  } = useQuery<AllInternalGroupsReturns>(ALL_INTERNAL_WORK_GROUPS)
-  const {
-    loading: interestLoading,
-    error: interestError,
-    data: interestData,
-  } = useQuery<AllInterestGroupsReturns>(ALL_INTEREST_GROUPS)
 
-  if (internalError) return <FullPageError />
-  if (internalLoading || !internalData) return <FullContentLoader />
+  const allInternalGroupTypesQuery = useQuery<AllInternalGroupsReturns>(
+    ALL_INTERNAL_GROUPS_QUERY
+  )
 
-  if (interestLoading || !interestData) return <FullContentLoader />
+  if (allInternalGroupTypesQuery.error) return <FullPageError />
+  if (allInternalGroupTypesQuery.loading || !allInternalGroupTypesQuery.data)
+    return <FullContentLoader />
 
-  const { allInternalWorkGroups } = internalData
-  const { allInterestGroups } = interestData
+  const internalGroups = allInternalGroupTypesQuery.data?.internalGroups
+  const interestGroups = allInternalGroupTypesQuery.data?.interestGroups
 
   return (
     <Wrapper>
-      <InternalGroupsFlexContainer>
-        <InternalGroupsContainerTitle>Drift</InternalGroupsContainerTitle>
-        <InternalGroupsContainer>
-          {allInternalWorkGroups.map((group: InternalGroupNode) => (
-            <div
-              onClick={() => {
-                history.push(`/internal-groups/${group.id}`)
-              }}
-            >
-              <InternalGroupCard internalGroup={group} />
-            </div>
-          ))}
-        </InternalGroupsContainer>
-        <InternalGroupsContainerTitle>
-          Interessegrupper
-        </InternalGroupsContainerTitle>
-        <InternalGroupsContainer>
-          {allInterestGroups.map((group: InternalGroupNode) => (
-            <div
-              onClick={() => {
-                history.push(`/internal-groups/${group.id}`)
-              }}
-            >
-              <InternalGroupCard internalGroup={group} />
-            </div>
-          ))}
-        </InternalGroupsContainer>
-      </InternalGroupsFlexContainer>
+      <InternalGroupsContainerTitle>Drift</InternalGroupsContainerTitle>
+      <InternalGroupsContainer>
+        {internalGroups.map((group: InternalGroupNode) => (
+          <div
+            key={group.id}
+            onClick={() => {
+              history.push(`/internal-groups/${group.id}`)
+            }}
+          >
+            <InternalGroupCard internalGroup={group} />
+          </div>
```

Comment:
can't this be just the `InternalGroupCard`?

```suggestion
            <InternalGroupCard internalGroup={group} 
                key={group.id}
                onClick={() => {
                    history.push(`/internal-groups/${group.id}`)
                 }}
            />
```

Reply:
pushed the onClick function onto InternalGroupCard.tsx instead, works fine. <img src="https://media1.giphy.com/media/QaN5BogCfpp6hQ4GA9/giphy.gif"/>